### PR TITLE
Feat db pools master sync

### PR DESCRIPTION
**Pull request by lohanidamodar**
## What does this PR do?

(Provide a description of what this PR does.)

## Test Plan

(Write your test plan here. If you changed any code, please provide us with clear instructions on how you verified your changes work.)

## Related PRs and Issues

(If this PR is related to any other PR or resolves any issue or related to any issue link all related PR and issues here.)

### Have you added your change to the [Changelog](https://github.com/appwrite/appwrite/blob/master/CHANGES.md)?

(The CHANGES.md file tracks all the changes that make it to the `main` branch. Add your change to this file in the following format)
- One line description of your PR [#pr_number](Link to your PR)

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

(Write your answer here.)

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -478,8 +480,8 @@ services:
       - _APP_EXECUTOR_SECRET
       - _APP_EXECUTOR_HOST
       - _APP_USAGE_STATS
-      - DOCKERHUB_PULL_USERNAME
-      - DOCKERHUB_PULL_PASSWORD
+      - APP_DOCKER_HUB_PULL_USERNAME
+      - APP_DOCKER_HUB_PULL_PASSWORD
```
Comment (abnegate) [inline docker-compose.yml:485]: Not directly related to this PR but these vars should be prefixed with `_`

Comment (lohanidamodar) [reply docker-compose.yml]: resolved

**Review by abnegate — APPROVED** [review]

File: .gitmodules [hunk 2]
```diff
@@ -1,4 +1,4 @@
 [submodule "app/console"]
 	path = app/console
 	url = https://github.com/appwrite/console
-	branch = db-pools-support
+	branch = 2.2.0
```
Comment (christyjacob4) [inline .gitmodules:4]: the 2.2.0 tag is not compatible with the db-pools branch yet. Please check with @TorstenDittmann / @ArmanNik  / @TGlide  on the status

File: .env [hunk 3]
```diff
@@ -67,6 +67,10 @@ _APP_USAGE_STATS=enabled
...
 _APP_LOGGING_CONFIG=
-_APP_REGION=default
+_APP_GRAPHQL_MAX_BATCH_SIZE=10
+_APP_GRAPHQL_MAX_COMPLEXITY=250
+_APP_GRAPHQL_MAX_DEPTH=3
 _APP_DOCKER_HUB_USERNAME=
-_APP_DOCKER_HUB_PASSWORD=
\ No newline at end of file
+_APP_DOCKER_HUB_PASSWORD=
+_APP_DOCKER_HUB_EMAIL=
```
Comment (christyjacob4) [inline .env:75]: looks like this is not used anywhere.

File: app/workers/mails.php [hunk 4]
```diff
@@ -32,67 +29,12 @@ public function run(): void
...
-                $body->setParam('{{team}}', $team->getAttribute('name'));
-                break;
-            case MAIL_TYPE_RECOVERY:
-            case MAIL_TYPE_VERIFICATION:
-            case MAIL_TYPE_MAGIC_SESSION:
-                $subject = $locale->getText("$prefix.subject");
-                break;
-            default:
-                throw new Exception('Undefined Mail Type : ' . $type, 500);
-        }
```
Comment (christyjacob4) [inline app/workers/mails.php:78]: Do you know why all this was removed? Can you point me to the PR ?

**Review by christyjacob4 — COMMENTED** [review]
Looks good. Just some minor comments

File: app/workers/mails.php (hunk 4 above)
Comment (lohanidamodar) [reply app/workers/mails.php:78]: @christyjacob4 here you go
https://github.com/appwrite/appwrite/pull/4859

File: .env (hunk 3 above)
Comment (lohanidamodar) [reply .env]: Fixed

File: .gitmodules (hunk 2 above)
Comment (lohanidamodar) [reply .gitmodules:4]: Posted message in PR review channel.

Comment (christyjacob4) [reply .gitmodules:4]: @lohanidamodar you can use the `feat-cloud` branch and change the build arg (VITE_CONSOLE_MODE I think ) in the docker-compose file to `cloud`

**Pull request merged**